core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
This is expected from the prompt_path function. Can you try `wh_media_util:get_prompt(<<"vm-saved">>).`? On mine I get `<<"prompt://system_media/vm-saved/en-us">>` which is the Kazoo-encoded prompt path.

For testing what media_mgr should return to ecallmgr, you can use:

```erlang
media_mgr_maintenance:prompt_url(<<"vm-saved">>).
media_mgr_maintenance:prompt_url(<<"vm-saved">>, <<"{ACCOUNT_ID}">>).
media_mgr_maintenance:prompt_url(<<"vm-saved">>, <<"{ACCOUNT_ID}">>, <<"la-ng">>).
```
I'll do a diff between 3.18 and master for the media_mgr app and the whistle_media core lib and see if there's an obvious difference that could account for the different results you and I are seeing.